HappyFox Test Automation Task Java /HelpDeskProductPageObject/src/main/java/CustomerPortal/SupportPortalPage.java

```diff
@@ -0,0 +1,131 @@
+package CustomerPortal;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import Admin_Portal.AdminPortalTest2ndPage;
+import Admin_Portal.Login_Page;
+import webdriverbase.AppPage;
+
+
+public class SupportPortalPage extends AppPage{
+	
+	public SupportPortalPage(WebDriver driver)
+	{
+		super(driver);
+				
+	}
+	
+	
+	public void navigateToHappyFoxSupportPortalURL(String url) {
+		this.driver.get(url);
+	}
+		
+	@FindBy(id = "id_subject")  

```

Comment:
Al elements should be declared first and then following methods should be there for more readable and maintain